scripts/actions/spells/trust/lion.lua

```diff
@@ -13,6 +13,21 @@ end
 
 spellObject.onMobSpawn = function(mob)
     xi.trust.message(mob, xi.trust.messageOffset.SPAWN)
+
+    -- Stun all the things!
+    mob:addSimpleGambit(ai.t.TARGET, ai.c.READYING_WS, 0,
+                        ai.r.MA, ai.s.SPECIFIC, 3198) -- Grape Shot (Stun)
+
+    mob:addSimpleGambit(ai.t.TARGET, ai.c.READYING_MS, 0,
+                        ai.r.MA, ai.s.SPECIFIC, 3198) -- Grape Shot (Stun)
+
+    mob:addSimpleGambit(ai.t.TARGET, ai.c.READYING_JA, 0,
+                        ai.r.MA, ai.s.SPECIFIC, 3198) -- Grape Shot (Stun)
+
+    mob:addSimpleGambit(ai.t.TARGET, ai.c.CASTING_MA, 0,
+                        ai.r.MA, ai.s.SPECIFIC, 3198) -- Grape Shot (Stun)
+
+    mob:setTrustTPSkillSettings(ai.tp.OPENER, ai.s.RANDOM, 1000)
```

Comment:
https://www.bg-wiki.com/ffxi/Cipher:_Lion

```
Uses TP as soon as she gets it.
```
What does she do on retail?

```lua
mob:setTrustTPSkillSettings(ai.tp.ASAP, ai.s.RANDOM)
```

Reply:
not sure why CI is now failing for unused variable.  Noticed Monberaux is using magic numbers as well?
```
    mob:addSimpleGambit(ai.t.CASTER, ai.c.MPP_LT, 25, ai.r.MS, ai.s.SPECIFIC, 4254, healingMoveCooldown) -- Mix: Dry Ether Concoction
    mob:addSimpleGambit(ai.t.PARTY, ai.c.HPP_LT, 90, ai.r.MS, ai.s.SPECIFIC, 4236, healingMoveCooldown) -- Max Potion (500 HP)
    mob:addSimpleGambit(ai.t.PARTY, ai.c.HPP_LT, 75, ai.r.MS, ai.s.SPECIFIC, 4237, healingMoveCooldown) -- Mix: Max Potion (700 HP)
 ```

Reply:
```
scripts/actions/spells/trust/lion.lua:18:11: (W211) unused variable kGrapeshot
    scripts/actions/spells/trust/lion.lua:23:83: (W113) accessing undefined variable kGrapeShot
    scripts/actions/spells/trust/lion.lua:24:83: (W113) accessing undefined variable kGrapeShot
    scripts/actions/spells/trust/lion.lua:[25](https://github.com/LandSandBoat/server/actions/runs/7348162533/job/20005774806?pr=4912#step:12:26):83: (W113) accessing undefined variable kGrapeShot
    scripts/actions/spells/trust/lion.lua:[26](https://github.com/LandSandBoat/server/actions/runs/7348162533/job/20005774806?pr=4912#step:12:27):83: (W113) accessing undefined variable kGrapeShot
```

kGrapeshot != kGrapeShot

> Noticed Monberaux is using magic numbers as well?

Are you asking why his local var doesn't throw up any warnings (there isn't a typo)? Or why he still has some magic numbers floating around (we're improving the codebase over time)?

Reply:
Reminder that's lots of existing code can be made better and if its existence was a reason not to, we would still look like DSP

Sometimes even I am naughty and do things without noticing that should be updated without noticing I did so.

Reply:
> ```
> scripts/actions/spells/trust/lion.lua:18:11: (W211) unused variable kGrapeshot
>     scripts/actions/spells/trust/lion.lua:23:83: (W113) accessing undefined variable kGrapeShot
>     scripts/actions/spells/trust/lion.lua:24:83: (W113) accessing undefined variable kGrapeShot
>     scripts/actions/spells/trust/lion.lua:[25](https://github.com/LandSandBoat/server/actions/runs/7348162533/job/20005774806?pr=4912#step:12:26):83: (W113) accessing undefined variable kGrapeShot
>     scripts/actions/spells/trust/lion.lua:[26](https://github.com/LandSandBoat/server/actions/runs/7348162533/job/20005774806?pr=4912#step:12:27):83: (W113) accessing undefined variable kGrapeShot
> ```
> 
> kGrapeshot != kGrapeShot
> 
> > Noticed Monberaux is using magic numbers as well?
> 
> Are you asking why his local var doesn't throw up any warnings (there isn't a typo)? Or why he still has some magic numbers floating around (we're improving the codebase over time)?

thank you, fixed.
my question was answered :)

Reply:
> Reminder that's lots of existing code can be made better and if its existence was a reason not to, we would still look like DSP
> 
> Sometimes even I am naughty and do things without noticing that should be updated without noticing I did so.

no, no, this wasn't some double-standard argument, i was wondering if the CI was failing for some reason due to trying to use a variable in that space, turns out i just typod the var name.

Reply:
I can see missing that on reading the CI log at first.